doc/source/whatsnew/v1.3.0.rst

```diff
@@ -715,6 +716,63 @@ Build
 
 Deprecations
 ~~~~~~~~~~~~
+- Deprecated allowing scalars to be passed to the :class:`Categorical` constructor (:issue:`38433`)
```

Comment:
hmm you have a rebase issue. only *your* changes should show here (this section was moved so think adding it back from where it was)